schema.graphql

```diff
@@ -152,7 +152,7 @@ type TweetInteraction {
 }
 
 type TweetMedia {
-  images: [String!]
+  images: [Shape!]
```

Comment:
What is a `Shape` and where is it's `type` defined? I don't see it in the schema and, naively, it seems like a surprising choice of model for an asset URL.

Reply:
I thought that was the type for an array or nested data, I tried that to get rid of the linter error but it did not so I was probably misunderstanding that. Trying out more fixes now

Reply:
FYI - `schema.graphql` and `schema.json` are generated files from `rake graphql:schema:dump`. This would be a good thing to include in the README.md in a separate PR: https://www.pivotaltracker.com/story/show/178305541

Reply:
To elaborate a little more, these files represent the graphql schema that is defined in the rails server. The `schema.json` file is used by eslint-graphql to check the graphql queries in the javascript files.